java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2550,6 +2552,56 @@ public T touch(@NonNull String[] keys) {
         return getThis();
     }
 
+    /**
+     * Counts the number of set bits (population counting) in a string of specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @return The number set bits in the string.
+     */
+    public T bitcount(@NonNull String key) {
+        ArgsArray commandArgs = buildArgs(key);
+        protobufTransaction.addCommands(buildCommand(Bitcount, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Counts the number of set bits (population counting) in a string of specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @param start The starting offset.
+     * @param end The ending offset.
+     * @return The number set bits in the string.
+     */
+    public T bitcount(@NonNull String key, long start, long end) {
+        ArgsArray commandArgs =
+                buildArgs(ArrayUtils.addAll(new String[] {key, Long.toString(start), Long.toString(end)}));
+
+        protobufTransaction.addCommands(buildCommand(Bitcount, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Counts the number of set bits (population counting) in a string of specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @param start The starting offset.
+     * @param end The ending offset.
+     * @param options The index offset type. Options are BYTE index or BIT index.
+     * @return The number set bits in the string.
+     */
+    public T bitcount(@NonNull String key, long start, long end, BitcountOptions options) {
+        ArgsArray commandArgs =
+                buildArgs(
+                        ArrayUtils.addAll(
+                                new String[] {key, Long.toString(start), Long.toString(end)}, options.toArgs()));
```

Comment:
Could be simplified everywhere (in client too)
```suggestion
                buildArgs(key, Long.toString(start), Long.toString(end), options.toString()));
```

Reply:
I think I've addressed where needed, let me know if I missed something though. Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00

Reply:
you can skip `addAll`, it is no-op